ansible/playbooks/AdoptOpenJDK_AIX_Playbook/main.yml

```diff
@@ -11,6 +11,8 @@
     PERL_MB_OPT: "--install_base /opt/freeware"
     PERL_MM_OPT: "INSTALL_BASE=/opt/freeware"
 
+  force_handlers: True
```

Comment:
Can this be localised into the security role or does it have to be in the top level yml file?

Reply:
Handlers are triggered when plays succeed and due to role-based directory organisation there's no other place it can be put and since there some warnings and errors, the handler is unlikely to be run. Also, this is the first handler in the entire playbooks collection, so it's not affecting other handlers. We can move it to ansible-playbook cli, though as the option " --force-handlers"

Reply:
To be clear, I'm OK with the original line in the top level yml or the documentation - just wanted to check if it was possible to move it to the lower level, andit sounds like that's a "no" :-)